### ISSUE TITLE
Latest Linux Build no longer includes LASzip - breaking GH actions tests that use PDAL writers.las with compression

### ISSUE THREAD

**Issue by rosepearson** [conda-forge/pdal-feedstock#145]
- [x] I read [the conda-forge documentation](https://conda-forge.org/docs/user/introduction.html#how-can-i-install-packages-from-conda-forge) and could not find the solution for my problem there.

Issue: I've been using GitHub actions to setup an invironment that contains PDAL and run some tests. Some of the tests use PDAL writers.las with the laszip option selected. I am now getting the error:

```
>   ???
E   RuntimeError: writers.las: Can't write LAZ output. PDAL not built with LASzip.
```

I did not used to get this error. As far as I can tell LASzip was removed from the PDAL build process in either  linux-ppc64le/pdal-2.3.0-ha6ad991_10.tar.bz2 or linux-aarch64/pdal-2.3.0-ha0ab71b_10.tar.bz2 as my tests ran without this error prior to that release.

I'm also a bit new to feedstocks, so I'm sorry if this is intended behaviour, or not the right place to be reporting this bug/change in behaviour.

You can see my tests + the package install specifics at:
No error: https://github.com/rosepearson/GeoFabrics/runs/4112504965?check_suite_focus=true
Error: **https://github.com/rosepearson/GeoFabrics/runs/4133556866?check_suite_focus=true**

<br/>
Environment (<code>conda list</code>):
<details>

Checkout: https://github.com/rosepearson/GeoFabrics/blob/main/environment_linux.yml
</details>

<br/>
Details about  <code>conda</code> and system ( <code>conda info</code> ):
<details>

https://github.com/rosepearson/GeoFabrics/blob/main/tests/test_processor_local_files/test_processor_local_files.py

Checkout line 127-136. 

```
        pdal_pipeline_instructions = [
            {"type":  "writers.las",
             "a_srs": f"EPSG:{cls.instructions['instructions']['output']['crs']['horizontal']}+" +
             f"{cls.instructions['instructions']['output']['crs']['vertical']}",
             "filename": str(lidar_file),
             "compression": "laszip"}
        ]

        pdal_pipeline = pdal.Pipeline(json.dumps(pdal_pipeline_instructions), [lidar_array])
        pdal_pipeline.execute()
```

The error results from `"compression": "laszip"` for `writers.las`
</details>

**Comment by hobu** [comment]
I quite likely broke something in #144 that was just merged today. We never did have linux ppc64le  or aarch64 prior to implementation of #144. As part of that, I switched LASzip for laz-perf.

Pin yourself back to build 9 for now, and I'll try to fix this.

**Comment by hobu** [comment]
Should be fixed in #146

**Comment by rosepearson** [comment]
Thanks @hobu! I just ran through now. It works a treat. Thanks for the quick fix!
